### PR TITLE
BAM-3713: Merchant Jstag documentation update

### PR DESCRIPTION
**Pull request by nickcroce**

### CODE REVIEW

File: docs/tagmerchant.rst [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
+Narrativ Merchant Tag
 =======================
 
 Functionality
 -------------
 
-The Narrativ advertiser tag allows Narrativ to track user behavior on an advertiser's site. Our auction system 
-optimizes advertiser acquisition and ROI by analyzing data on page views and purchase behavior. No Personal 
-Identifying Information (PII) is captured. The tag runs asynchronously in the background so there is no impact to page 
+The Narrativ merchant tag allows Narrativ to track user behavior on an merchant's site. Our auction system
```
Comment (myfashionhub) [inline docs/tagmerchant.rst]: " a merchant's"

File: docs/tagmerchant.rst [hunk 2]
```diff
@@ -110,10 +110,12 @@ just leave it out if you choose not to include the data.
      - string
      - Optional. The color of the product.
 
-Full Example
+**Full Example**
 
-What you see below is just an example. You cannot copy and paste it as is. Insert the check out
-information from your own page into our data layer using Javascript before loading the Narrativ tag. Remember to replace "accountname" with your Narrativ account name.
+Below is an example of what your implementation might look like. However, don't copy and paste it as is. Insert the checkout
```
Comment (kaitlynkrull) [inline docs/tagmerchant.rst]: "The example below is what your implementation might look like" sounds better to me

File: docs/tagmerchant.rst [hunk 3]
```diff
@@ -165,10 +167,10 @@ information from your own page into our data layer using Javascript before loadi
 Adding Information for the Page View Tag
 ----------------------------------------
 
-The page view tag goes on every page except for pages with PII. Remember to change "ACCOUNT NAME" to your Narrativ 
+The page view tag goes on every page except for pages with PII. Remember to change "ACCOUNT NAME" to your Narrativ
 account name.
 
-What you see below is another example. You cannot copy and paste it as is. Insert the page view
+Below is another example of what your implementation might look like. Don't copy and paste it as is. Insert the page view
```
Comment (kaitlynkrull) [inline docs/tagmerchant.rst]: same here

File: docs/tagmerchant.rst [hunk 4]
```diff
@@ -1,36 +1,36 @@
...
 
-* Place the check out tag on the page that loads after your customer has finished shopping and paid
+* Place the checkout tag on the page that loads *after* your customer has finished shopping and paid
   for their purchase (your site's equivalent of a "Thank you for your order" page).
 
 * Place the page view tag on every other page that loads during your user's journey, so that we can
-  empower you with data on what products they are looking at before they check out.
-  Important! Replace "ACCOUNT NAME" with your Narrativ account name in all snippets. Reach out to
+  empower you with data on what products customers are looking at before they check out.
+  *Important! Replace "ACCOUNT NAME" with your Narrativ account name in all snippets.* Reach out to
```
Comment (myfashionhub) [inline docs/tagmerchant.rst]: can we make `"ACCOUNT NAME"` `"accountname"` throughout? Slug is lowercase with no space

File: docs/tagmerchant.rst [hunk 5]
```diff
@@ -191,15 +193,17 @@ information into our data layer using Javascript before loading the Narrativ tag
 Google Tag Manager Walkthrough
 ------------------------------
 
-Below is a walkthrough for implementing the Narrativ pixel using a "Custom HTML" tag in Google Tag Manager.
+Below are instructions for implementing the Narrativ pixel using a "Custom HTML" tag in Google Tag Manager.
```
Comment (kaitlynkrull) [inline docs/tagmerchant.rst]: The instructions below are for implementing the Narrativ pixel using a "Custom HTML" tag in Google Tag Manager

File: docs/tagmerchant.rst [hunk 6]
```diff
@@ -191,15 +193,17 @@ information into our data layer using Javascript before loading the Narrativ tag
...
 Start by navigating to your Google Tag Manager Dashboard.
 
 
-- Select "Tag Configuration" to begin making a new pixel.
+- Select the "Tags" menu item from the menu on the left-hand side of the page.
+- Once in the "Tags" menu, select the "New" button to create a new tag.
```
Comment (myfashionhub) [inline docs/tagmerchant.rst]: Once on "Tags" page?

File: docs/tagmerchant.rst [hunk 7]
```diff
@@ -1,36 +1,36 @@
...
 Functionality
 -------------
 
-The Narrativ advertiser tag allows Narrativ to track user behavior on an advertiser's site. Our auction system 
-optimizes advertiser acquisition and ROI by analyzing data on page views and purchase behavior. No Personal 
-Identifying Information (PII) is captured. The tag runs asynchronously in the background so there is no impact to page 
+The Narrativ merchant tag allows Narrativ to track user behavior on an merchant's site. Our auction system
+optimizes merchant acquisition and ROI by analyzing data on page views and purchase behavior. No Personal
+Identifying Information (PII) is captured. The tag runs asynchronously in the background so there is no impact to page
 load times.
```
Comment (ChrisCavs) [inline docs/tagmerchant.rst]: break line 9 after `impact`, and have line 10 be `to page load times`

File: docs/tagmerchant.rst [hunk 8]
```diff
@@ -1,36 +1,36 @@
...
 
 Adding Information About Your Checkout Page
 -------------------------------------------
 
 
 For checkout events, you have to fill in some information about the checkout before loading our Javascript. This is
-done by inserting the relevant data into a variable called window.BAMX_EVENT_DATA. If a field is optional, you can
-just leave it out if you choose not to include the data.
+done by inserting the relevant data into a variable called ``window.BAMX_EVENT_DATA``. If a field is optional, you can
+leave it out if you choose not to include the data.
```
Comment (ChrisCavs) [inline docs/tagmerchant.rst]: break line 30 after 'you', add `can leave` to line 31

**Comment by nickcroce** [comment]
Should this have a version update in `conf.py`?

**Pull request merged**